librz/bin/bin.c

```diff
@@ -649,6 +649,14 @@ RZ_API RzList *rz_bin_get_sections(RzBin *bin) {
 	return o ? (RzList *)rz_bin_object_get_sections_all(o) : NULL;
 }
 
+/**
+ * \brief Find the binary section at offset \p off.
+ *
+ * \param o Reference to the \p RzBinObject instance
+ * \param off Address to search
+ * \param va When 0 the offset \p off is considered a physical address, otherwise a virtual address
+ * \return Pointer to a \p RzBinSection containing the address
+ */
 RZ_API RzBinSection *rz_bin_get_section_at(RzBinObject *o, ut64 off, int va) {
```

Comment:
Please add also an ownership mark on the return.

Reply:
added to the .h file, like the others